src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,15 @@
 import React from 'react';
+import { Posts } from '../../types/Posts';
+import { PostInfo } from '../PostInfo';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  posts: Posts[],
```

Comment:
I think better names for Type would be Post, Comment and User. Because you use it for only one object)